src/main/java/dk/cngroup/lentils/service/SearchService.java

```diff
@@ -0,0 +1,65 @@
+package dk.cngroup.lentils.service;
+
+import dk.cngroup.lentils.entity.Team;
+import org.apache.lucene.search.Query;
+import org.hibernate.search.jpa.FullTextEntityManager;
+import org.hibernate.search.jpa.Search;
+import org.hibernate.search.query.dsl.QueryBuilder;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import javax.persistence.EntityManager;
+import javax.persistence.NoResultException;
+import java.util.List;
+
+@Service
+public class SearchService {
+
+    @Autowired
+    private final EntityManager centityManager;
```

Comment:
Typo: centity -> entity?

Reply:
je to schvalne - abych to rozlisil (stejne to mam v SearchConfiguration :P
odstranim ;)

Reply:
V tom pripade, to prosim rozlis nejak jinak, protoze ja jsem si myslel u `bentity`, ze je to taky typo. Btw: myslim si, ze se muzou jmenovat klidne stejne.